persistence/proto/block_persistence.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package persistence;
+
+option go_package = "github.com/pokt-network/pocket/persistence/types";
+
+message Block {
```

Comment:
Transactions were planned to be stored in the `transaction indexer` is there a reason you think we should store them in a `Block`?

Consider `TransactionsHash` to represent them instead